dashboard/src/features/projects/custom-domains/settings/components/AuthDomain/AuthDomain.tsx

```diff
@@ -0,0 +1,155 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useEffect, useState } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { toast } from 'react-hot-toast';
+import * as Yup from 'yup';
+
+const validationSchema = Yup.object({
+  auth_fqdn: Yup.string().required(),
+});
+
+export type AuthDomainFormValues = Yup.InferType<typeof validationSchema>;
+
+export default function AuthDomain() {
+  const { maintenanceActive } = useUI();
+  const [isVerified, setIsVerified] = useState(false);
+  const { currentProject, refetch: refetchWorkspaceAndProject } =
+    useCurrentWorkspaceAndProject();
+
+  const [updateConfig] = useUpdateConfigMutation();
+
+  const form = useForm<{ auth_fqdn: string }>({
+    reValidateMode: 'onSubmit',
+    defaultValues: { auth_fqdn: null },
+    resolver: yupResolver(validationSchema),
+  });
+
+  const { data, loading, error } = useGetAuthenticationSettingsQuery({
+    variables: {
+      appId: currentProject.id,
+    },
+  });
+
+  useEffect(() => {
+    if (!loading && data) {
+      const { networking } = data?.config?.auth?.resources || {};
+      const fqdn = networking?.ingresses?.[0]?.fqdn?.[0];
+      form.reset({ auth_fqdn: fqdn });
+    }
+  }, [data, loading, form]);
+
+  if (loading) {
+    return (
+      <ActivityIndicator
+        delay={1000}
+        label="Loading Auth Domain Settings..."
+        className="justify-center"
+      />
+    );
+  }
+
+  if (error) {
+    throw error;
+  }
+
+  const { formState, register, watch } = form;
+  const isDirty = Object.keys(formState.dirtyFields).length > 0;
+
+  const auth_fqdn = watch('auth_fqdn');
+
+  async function handleSubmit(formValues: AuthDomainFormValues) {
+    const updateConfigPromise = updateConfig({
+      variables: {
+        appId: currentProject.id,
+        config: {
+          auth: {
+            resources: {
+              networking: {
+                ingresses: [
+                  {
+                    fqdn: [formValues.auth_fqdn],
+                  },
+                ],
+              },
+            },
+          },
+        },
+      },
+    });
+
+    try {
+      await toast.promise(
+        updateConfigPromise,
+        {
+          loading: `Auth domain is being updated...`,
+          success: `Auth domain has been updated successfully.`,
+          error: getServerError(
+            `An error occurred while trying to update the auth domain.`,
+          ),
+        },
+        getToastStyleProps(),
+      );
+
+      form.reset(formValues);
+      await refetchWorkspaceAndProject();
+    } catch {
+      // Note: The toast will handle the error.
+    }
+  }
+
+  return (
+    <FormProvider {...form}>
+      <Form onSubmit={handleSubmit}>
+        <SettingsContainer
+          title="Auth Domain"
+          description="It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout."
```

Comment:
This has to be removed or replaced by a correct description

Reply:
@nunopato Can you help out the copy?